src/main.rs

```diff
@@ -30,65 +42,85 @@ async fn main() -> Result<(), Error> {
 
     yansi::whenever(Condition::cached((USE_COLOR)()));
 
-    for url in &args.urls {
-        println!("Getting: {:?}", url);
-        println!(
-            "{} {}Validating url...",
-            "[1/3]".bold().yellow(),
+    let url_count = args.urls.len();
+    for (i, url) in args.urls.iter().enumerate() {
+        log::info!(
+            "{} Cloning {url:?}...",
+            format!("[{}/{}]", i + 1, url_count + 1).bold().blue()
+        );
+
+        let steps = 3 + (if args.zipped { 2 } else { 0 });
```

Comment:
This feels kinda silly but unless we have a better way of organizing each "step" this will have to do.

Reply:
True, would need a to re-structure the code for better way..